hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -203,6 +206,25 @@ private void encodeAndWriteParityCells() throws IOException {
     }
   }
 
+  private void logStreamError(List<ECBlockOutputStream> failedStreams,
+                              String operation) {
+    Set<Integer> failedStreamIndexMap =
+            failedStreams.stream().map(ECBlockOutputStream::getReplicationIndex)
+                    .collect(Collectors.toSet());
+
+    String failedStreamsString = IntStream.range(1,
+                    numDataBlks + numParityBlks + 1)
+            .mapToObj(index -> failedStreamIndexMap.contains(index)
+                    ? "F" : "S")
+            .collect(Collectors.joining(" "));
+    LOG.error("{} failed: {}", operation, failedStreamsString);
```

Comment:
In EC we will actually proceed by closing current blockGroup. Should we change the log level to warn instead?